app/models/knight.rb

```diff
@@ -1,5 +1,23 @@
 class Knight < Piece
   scope :piece, -> {where(type: 'Knight')}
 
+  def valid_move?(new_x, new_y)
+    return false if outside_board?(new_x, new_y)
+    #does not matter if knight is obstructed on its way to its new space; it can jump over other pieces
+    #I know this code is about as dry as the ocean, so refactoring suggestions are appreciated! :)
+    
+    #knight can move up 2, left or right 1
+    return true if self.position_x + 1 == new_x && self.position_y + 2 == new_y
```

Comment:
@drewsef-916 the logic looks solid.. lets try to refactor the code a little bit.. a few ideas could be:
Lets take a move of 2 up and 1 left or right... is there a way we can express it in a concise way? what is the difference between moving `2 up and 1 left` and `2 up and 1 right`.. once you follow that path the whole method can be refactored really nicely.. hope it makes sense